backend/test/integrated/teamspace.js

```diff
@@ -366,4 +373,304 @@ describe("Checking Quota Info  ", function() {
+		{ "value" : "safety_issue_-_fall", "label" : "Safety Issue - Fall" },
+		{ "value" : "safety_issue_-_trapped", "label" : "Safety Issue - Trapped" },
+		{ "value" : "safety_issue_-_event", "label" : "Safety Issue - Event" },
+		{ "value" : "safety_issue_-_handling", "label" : "Safety Issue - Handling" },
+		{ "value" : "safety_issue_-_struck", "label" : "Safety Issue - Struck" },
+		{ "value" : "safety_issue_-_public", "label" : "Safety Issue - Public" },
+		{ "value" : "social_issue", "label" : "Social Issue" },
+		{ "value" : "other_issue", "label" : "Other Issue" },
+		{ "value" : "unknown", "label" : "UNKNOWN" }
+	];
+	const defaultRiskCategoryLabels = defaultRiskCategories.map(x => x.label);
+	const defaultTopicTypes =  [
+		{ "value" : "for_information", "label" : "For information" },
+		{ "value" : "vr", "label" : "VR" }
+	];
+	const defaultTopicTypeLabels = defaultTopicTypes.map(x => x.label);
+
+	describe("Update teamspace settings", function(done) {
+		const user =  imsharedTeamspace;
+		const newRiskCategories = [
+			{ "value": "new_cat_1", "label": "New Cat 1" },
+			{ "value": "new_cat_2", "label": "New Cat 2" }
+		];
+		const newRiskCategoryLabels = newRiskCategories.map(x => x.label);
+		const newTopicTypes = [
+			{ "value": "new_type_1", "label": "New Type 1" },
+			{ "value": "new_type_2", "label": "New Type 2" }
+		];
+		const newTopicTypeLabels = newTopicTypes.map(x => x.label);
+
+		before(function(done) {
+			this.timeout(timeout);
+			agent.post("/login")
+				.send({username: user.user, password: user.password})
+				.expect(200, done);
+
+		});
+
+		it("set defaults should succeed", function(done) {
+			agent.put(`/${user.user}/settings`)
+				.send({ topicTypes: defaultTopicTypeLabels, riskCategories: defaultRiskCategoryLabels })
+				.expect(200, function(err, res) {
+					expect(res.body._id).to.equal(user.user);
+					expect(res.body.riskCategories).to.deep.equal(defaultRiskCategories);
+					expect(res.body.topicTypes).to.deep.equal(defaultTopicTypes);
+					done(err);
+				});
+		});
+
+		it("with new topic types should succeed", function(done) {
+			agent.put(`/${user.user}/settings`)
+				.send({ topicTypes: newTopicTypeLabels })
+				.expect(200, function(err, res) {
+					expect(res.body._id).to.equal(user.user);
+					expect(res.body.riskCategories).to.deep.equal(defaultRiskCategories);
+					expect(res.body.topicTypes).to.deep.equal(newTopicTypes);
+					done(err);
+				});
+		});
+
+		it("with new risk categories should succeed", function(done) {
+			agent.put(`/${user.user}/settings`)
+				.send({ riskCategories: newRiskCategoryLabels })
+				.expect(200, function(err, res) {
+					expect(res.body._id).to.equal(user.user);
+					expect(res.body.riskCategories).to.deep.equal(newRiskCategories);
+					expect(res.body.topicTypes).to.deep.equal(newTopicTypes);
+					done(err);
+				});
+		});
+
+		it("with unexpected field should succeed", function(done) {
```

Comment:
We need to check the unexpected field doesn't get added into the teamspace setting on the expect part (I'm pretty sure that's broken on staging but it's fixed in #1966 )